conf/modules/geo_mag.xml

```diff
@@ -4,8 +4,14 @@
   <doc>
     <description>
       GeoMagnetic field vector.
-      Calculation of the normalized geomagnetic field vector (saved to ahrs_impl.mag_h) at startup using GPS fix.
-      Based on the WMM2010 model (http://www.ngdc.noaa.gov/geomag/models.shtml).
+      Calculation of the normalized geomagnetic field vector (saved to ahrs_impl.mag_h) at startup as soon as a valid location is aquired.
+      Method uses is based on the the ngdc noaa DODWMM model and data (http://www.ngdc.noaa.gov/geomag/WMM/DoDWMM.shtml).       
+      To obtain the a magnetic heading to be used by the autopilot, the program must know the magnetic declination at all the points en route, at the indicated flight altitude. 
```

Comment:
Yes, my description skills need improvement. It was more as a brainwash for self to my still PPRZ TODO list to improve all module description to something more informative than some module description.